src/runtime/graph_executor/debug/graph_executor_debug.cc

```diff
@@ -114,15 +94,69 @@ class GraphExecutorDebug : public GraphExecutor {
 
     std::ostringstream os;
     for (size_t index = 0; index < time_sec_per_op.size(); index++) {
-      os << time_sec_per_op[index] << ",";
+      double time = time_sec_per_op[index];
+      // To have good behavior when calculating total time, etc.
+      if (isnan(time)) {
+        time = 0;
+      }
```

Comment:
I don't really understand how `time_sec_per_op` could become nan, but I this is probably a good change regardless.

Reply:
0 / 0 is possible from the above for nodes which do not have any associated execution function